docs/reference/elastic-search.md

```diff
@@ -117,7 +117,57 @@ deleted from the index. Attempts at reproducing this issue in a simpler
 environment have failed. As a workaround, there is a tool in
 [tools/db/find-undead](../../tools/db/find-undead) which can be used to find the
 undead users right after the migration. If they exist, please run refill the ES
-documents from cassandra as described [above](#refill-es-documents-from-cassandra)
+documents from cassandra as described [above](#refill-es-documents-from-cassandra**
+
+## Migrate to a new cluster
+
+If the ES cluster used by brig needs to be shutdown and data must be moved to a
+new cluser, these steps can be taken to ensure minimal disruption to the
+service.
+
+Before starting, please set these environment variables:
+
+```bash
+ES_OLD_HOST=<YOUR_HOST>
+ES_OLD_PORT=<YOUR_PORT> # usually 9200
+ES_OLD_INDEX=<INDEX_NAME_ALREADY_IN_USE>
+ES_NEW_HOST=<YOUR_HOST>
+ES_NEW_PORT=<YOUR_PORT> # usually 9200
+ES_NEW_INDEX=<NEW_INDEX_NAME>
+WIRE_VERSION=<VERSION_YOU_ARE_DEPLOYING>
+SHARDS=<NUMBER_OF_SHARDS_FOR_THE_NEW_INDEX>
+REPLICAS=<NUMBER_OF_REPLICAS_FOR_THE_INDEX>
+REFRESH_INTERVAL=<REFRESH_INTERVAL_IN_SECONDS>
```

Comment:
Can you document a command here which allows to see the current shards, replicas and refresh interval of the existing index (as I suppose one usually wants to use the same/similar settings)?

Reply:
I added a `curl` in the comment above the `SHARDS` line.